config/manager/manager.yaml

```diff
@@ -4,18 +4,18 @@ metadata:
   name: controller-manager
   namespace: system
   labels:
-    control-plane: controller-manager
+    control-plane: glance-controller-manager
```

Comment:
I just noticed one area where this particular change will cause us problems.  During CI we gather certain log/config data [1].  When we get the controller-manage logs, we select the pod via the `control-plane=controller-manager` label.  We might need to keep the old label but still add the new one.

https://github.com/gibizer/openshift-release/blob/7b87904de96e4b1f98d3b7a98cb4602c50500c09/ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh#L16

Reply:
Looks like regex is not supported for selectors[1], Unless we want to add a label for log collection, we change the command to 'oc get pods -n openstack --show-labels | awk '{if ($6 ~ "controller-manager") print $1}' | xargs -n1 oc logs'  in CI. I can propose a patch there if it's ok.

Reply:
Sorry, I missed your response here.  I think we should propose a patch for CI in this regard.

Reply:
agree with fixing in CI, it seems this is going to be required to make webhooks play nicely

Reply:
Pushed https://github.com/openshift/release/pull/36394